### ISSUE TITLE
Sort by album artist tag instead of artist tag

### ISSUE THREAD

**Issue by edw1nyang** [kabouzeid/phonograph-issue-tracker#10]
it'll make song listing less cluttered, like what google play music do. Like a song by Calvin Harris for example. Song title: How deep is your love, Artist: Calvin Harris ft. Disciples, Album artist: Calvin Harris. Sorting by artist will be pain because some artist, ex: Calvin Harris, in his single album he had done collaboration with many artists, so instead of having clutter like calvin harris ft. a, calvin Harris ft. b, calvin Harris & c, etc it'll be great if the sorting just showing calvin harris as the album artist. thank you in advance. your app have a ton of potential.

**Comment by kabouzeid** [comment]
This issue is well know. The problem here is that the Android Media Provider doesn't allow me to do that. It is really that simple. Google Play Music and PowerAMP are using custom Media Providers so they are one of the very few players which can achieve this behavior.

**Comment by edw1nyang** [comment]
issue reopened? fantastic! so are you gonna migrate to custom media providers as well? if so no reason for me to use gpm anymore :+1:

**Comment by kabouzeid** [comment]
@edw1nyang no, I just did that so other people won't post this issue again.
